packages/trip-details/src/index.tsx

```diff
@@ -54,27 +39,25 @@ function CO2DescriptionLink(contents: ReactElement): ReactElement {
     </a>
   );
 }
-
 /**
- * Default rendering if no component is provided for the CaloriesDetails
+ * Default rendering if no component is provided for the TimeActiveDetails
  * slot in the TripDetails component.
  */
-function DefaultCaloriesDetails({
-  bikeSeconds,
-  calories,
-  walkSeconds
-}: CaloriesDetailsProps): ReactElement {
+function DefaultTimeActiveDetails({
+  bikeDuration,
+  walkDuration
```

Comment:
We previously explicitly put the units in the prop names to remove any ambiguity, so `bikeDuration` and `walkDuration` should be renamed to `bikeMinutes` and `walkMinutes`.